index.html

```diff
@@ -0,0 +1,75 @@
+                        </path>
+                    </g>
+                </svg>
+            </div>
+            <div class="col text-right align-text-bottom">
+                <a id="a1" href="#">sign in</a>
+                <a id="a2" href="#">subscribe 
+                    <svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 8 14" style="enable-background:new 0 0 8 14;" xml:space="preserve"><g id="arrow-right">
+                        <defs>
+                        <rect id="SVGID_1_" width="8" height="14"></rect>
+                        </defs>
+                        <clipPath id="SVGID_2_">
+                        <use xlink:href="#SVGID_1_" style="overflow:visible;"></use>
+                        </clipPath>
+                        <g id="Shape_1" class="st0">
+                        <g>
+                        <polygon class="st1" points="8,7 1.1,0 0,1.1 5.9,7 0,12.9 1.1,14 8,7 8,7	"></polygon>
+                        </g>
+                        </g>
+                        </g>
+                    </svg>
+                </a>
+            </div>
+        </nav>
+        <div class="row header-bottom">
+        </div>
+        <div class="w-100"></div>
+        <div class="row" id="bar-with-anchors">
```

Comment:
**[Optional] This is not required for your project to be approved but I encourage you to consider.**
- [ ] Try using a  `<nav>` `<ul>` tag and `<li>`